LayoutTests/platform/ios/TestExpectations

```diff
@@ -3590,6 +3590,8 @@ webkit.org/b/239567 tables/mozilla/bugs/bug26178.html [ Pass Failure ]
 # iOS has a WebGPU implementation.
 http/tests/webgpu [ Pass ]
 
+webkit.org/b/240024 http/tests/quicklook/same-origin-xmlhttprequest-allowed.html [ Pass Failure Crash ]
```

Comment:
Why are you returning this line? This line was added in https://commits.webkit.org/250226@main which you are supposed to revert as well.

Reply:
Yes, that was an oversight. Deleted.